examples/ultrahdr_app.cpp

```diff
@@ -372,6 +374,7 @@ class UltraHdrAppInput {
   size_t mMapDimensionScaleFactor;
   int mMapCompressQuality;
   bool mUseMultiChannelGainMap;
+  float mGamma;
```

Comment:
const float mGainMapGamma; 
this keeps in sync with other constructor arg declarations